src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts

```diff
@@ -180,6 +187,7 @@ export class ViolationTicketService {
     let result = <ViolationTicket>{};
     let isDateFound = false;
     let isTimeFound = false;
+    this._systemDetectedOCRIssues = this.SystemDetectedOcrIssues.N;
```

Comment:
add checking inside keys.forEach and check if the key is inside the array or not